CDCgov/prime-reportstream
# Engagement/jessica/14613 indiana migration pre work

This PR implements the pre migration work for Indiana and updates the Simple Report sender transform, resulting in changes to the integration tests as well. 

Test Steps:
1. Review that the changes in organizations.yml match IN previous settings
2. Check that the resulting message matches the outbound message from the Covid pipeline
3. Make sure the tests pass

## Changes
- Added settings for Montana to organizations.yml
- Created the IN receiver transform
- Updated the simple report sender transform
- Updated documentation for the overall pre migration process
- Updated integration tests to reflect the changes in the sender transform

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
